mitmproxy/addons/allowremote.py

```diff
@@ -0,0 +1,16 @@
+import ipaddress
+from mitmproxy import ctx
+
+
+class AllowRemote:
+    def clientconnect(self, layer):
+        address = layer.client_conn.address
+
+        if not (ctx.options.allow_remote or
+                ipaddress.ip_address(address[0]).is_private or
```

Comment:
I think that's fine to be honest, the more interesting question is how this behaves with IPv6?